### ISSUE TITLE
Expose betas for AdamW to use with LR Scheduler

### ISSUE THREAD

**Issue by rockerBOO** [huggingface/candle#1448]
I am working on building out my training using Candle. Has been good so far but I missed some of my LR schedulers I was using. Specifically I have been building out OneCycleLR from https://arxiv.org/abs/1708.07120. PyTorch has an implementation [OneCycleLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html#torch.optim.lr_scheduler.OneCycleLR)

In OneCycleLR you modify both the LR and the momentum. For AdamW we can modify the beta1 value for the momentum. Would exposing the betas be invited? I can make a PR but proposing it first.

Here is the scheduler code I have so far [rockerBOO/candle_scheduler](https://github.com/rockerBOO/candle_scheduler). 

Thank you!

**Comment by LaurentMazare** [comment]
Sure, sounds like an easy and useful thing to do (probably exposing the whole `params` field would be fairly flexible).
